test/compilationTests/stringutils/strings.sol

```diff
@@ -83,23 +83,23 @@ library strings {
         uint ret;
         if (self == 0)
             return 0;
-        if (self & 0xffffffffffffffffffffffffffffffff == 0) {
+        if (self & bytes32(uint256(0xffffffffffffffffffffffffffffffff)) == 0) {
```

Comment:
This looks rather ugly, but yeah. Another fix would be to use `uint(self) & 0xfff...fff`, right?

By the way, could you also fix that upstream, please?

Reply:
Yes, at some point I noticed that casting the left hand side looks better, but I haven't yet changed all of them. I'll change it. "Upstream" for this is https://github.com/Arachnid/solidity-stringutils, right?

Reply:
Yes, I think so.